src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,201 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private RegistrationService registrationServiceImpl;
+
+    @BeforeEach
+    void setUp() {
+        registrationServiceImpl = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void registration_OneUserAdd_ok() {
+        User user1 = new User();
+        user1.setLogin("Terminator");
+        user1.setPassword("123456");
+        user1.setAge(18);
+        registrationServiceImpl.register(user1);
+        assertEquals(1, Storage.people.size());
+        assertEquals(user1, Storage.people.get(0));
+    }
+
+    @Test
+    void registration_twoUserAdd_ok() {
+        User user1 = new User();
+        user1.setLogin("Terminator");
+        user1.setPassword("123456");
+        user1.setAge(18);
+        User user2 = new User();
+        user2.setLogin("Vortex");
+        user2.setPassword("123456");
+        user2.setAge(18);
+        registrationServiceImpl.register(user1);
+        registrationServiceImpl.register(user2);
+        assertEquals(2, Storage.people.size());
+        assertEquals(user1, Storage.people.get(0));
+        assertEquals(user2, Storage.people.get(1));
+    }
```

Comment:
its enough to have 1 test for ok case
```suggestion
    }
```